scripts/in_container/run_resource_check.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+
+import psutil
+
+os.system("sh _in_container_script_init.sh")
```

Comment:
I  think we do not really need to source it - it was mainly needed for the bash script to source some common functions but we can simply drop it.

Reply:
Will these common functions ever get rewritten in python? or they are no longer necessary

Reply:
Not needed - mostly of the time (or we already use stuff instead - like "run_command"). Anything that will be needed will end up as a shared Python code.